Source/CesiumRuntime/Public/GlobeAwareDefaultPawn.h

```diff
@@ -112,11 +113,11 @@ class CESIUMRUNTIME_API AGlobeAwareDefaultPawn : public ADefaultPawn {
    * the flight interpolation.
    */
   UPROPERTY(
-      EditAnywhere,
-      BlueprintReadWrite,
-      Category = "Cesium",
-      meta = (ClampMin = 0.0))
-  float FlyToGranularityDegrees = 0.01f;
+      meta =
+        (DeprecatedProperty,
+         DeprecationMessage =
+               "FlyToGranularityDegrees has been deprecated. Flight interpolation is now always of the highest quality."))
```

Comment:
:) And I though that "Flight interpolation is now computed every frame" might be vague since technically, it did interpolate before as well, just between precalculated points. But in the end, I think the goal is to not make it confusing for the user. Essentially, don't think about it, just don't set this thing anymore.

Reply:
Yeah that's fair. I would have also suggested "Flight interpolation is now computed on the fly" but I thought the pun was unforgiveable. 😛